### PR TITLE
Improve pages table on receipt page

### LINKED ISSUE #47: Pages display broken on receipt
Discovered this while testing another issue.

screen shot 2016-09-19 at 12 37 38 pm

### PR DESCRIPTION
**Pull request by ghost1227**
Fixes #47

### CODE REVIEW

File: includes/email-tags.php [hunk 1]
```diff
@@ -45,8 +45,12 @@ function edd_cr_add_template_tags( $payment_id ) {
 	$page_list  = '<div class="edd_cr_accessible_pages">' . __( 'Pages', 'edd-cr' ) . '</div>';
 	$page_list .= '<ul>';
 
-	foreach ( $meta as $post ) {
-		$page_list .= '<li><a href="' . esc_url( get_permalink( $post->ID ) ) . '">' . $post->post_title . '</a></li>';
+	foreach ( $meta as $download_id => $pages ) {
```
Comment (pippinsplugins) [inline includes/email-tags.php]: Can you please go ahead and add `if( is_array( $meta ) )` here?

Comment (ghost1227) [inline includes/email-tags.php]: Wouldn't that be better suited up in the `if ( empty( $meta ) )` check? ie; it should be `if ( ! is_array( $meta ) || empty( $meta ) )`. We don't want the section to display at all if `$meta` isn't a list of available pages...

Comment (pippinsplugins) [inline includes/email-tags.php]: Just need to make sure we don't end up with a foreach on a non array error.

Comment (ghost1227) [inline includes/email-tags.php]: Fixed

**Pull request merged**